source/quick-start.txt

```diff
@@ -50,6 +50,12 @@ If you are using `Gradle <https://gradle.org/>`__, add the following to your
 
 .. include:: /includes/quick-start/gradle-versioned.rst
 
+.. tip:: Download ``.jar`` File
+   
+   You can directly download the ``.jar`` file for the {+driver-short+}
```

Comment:
Suggestion:
I think JAR is usually not referenced by it's filename extension, but rather by the file format "JAR". See https://en.wikipedia.org/wiki/JAR_(file_format)

E.g.
tip:: Download the JAR File